### PR TITLE
[4.0] Use namespaced version of JFactory everywhere

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

Replace `JFactory` with `Factory` in all remaining files

### CODE REVIEW

**Comment by brianteeman** [comment]
Finder was excluded when I did the big namespace conversion as it was still being rewritten by @Hackwar and I believe he has recently done it with #24097

**Comment by C-Lodder** [comment]
Reverted `com_finder`

**Comment by amitranjan2** [comment]
@C-Lodder I have seen somewhere JTEXT also 
could you please change that also?

**Comment by C-Lodder** [comment]
@amitranjan2 There's are loads of `J` prefixed classes that need changing. The PR focuses on solely on `JFactory`.

`JText` and the others can be done in separate PRs

**Comment by amitranjan2** [comment]
@C-Lodder ok I will do it my making another PR ,thanks

**Comment by amitranjan2** [comment]
@C-Lodder  are you sure that you have changed all `JFactory` to `Factory`?

I am seeing plenty of `JText`  here 
changed almost 50 JText  and still there are plenty

**Comment by C-Lodder** [comment]
Yes, I've changed all of the ones I found in a mass search, other than those in `com_finder` which I was asked to revert.

It would probably be much better creating a script that automatically does this.

**Comment by amitranjan2** [comment]
> Yes, I've changed all of the ones I found in a mass search, other than those in `com_finder` which I was asked to revert.
> 
> It would probably be much better creating a script that automatically does this.

that will be great,please create a script

File: administrator/components/com_content/tmpl/articles/default_batch_body.php [hunk 1]
```diff
@@ -9,10 +9,11 @@
 defined('_JEXEC') or die;
 
 use Joomla\CMS\Layout\LayoutHelper;
+use Joomla\CMS\Factory;
```
Comment (Quy) [inline administrator/components/com_content/tmpl/articles/default_batch_body.php:11]: Please sort in alpha order.

Comment (C-Lodder) [reply administrator/components/com_content/tmpl/articles/default_batch_body.php:11]: done, thanks

**Comment by dgrammatiko** [comment]
> that will be great,please create a script

There is no need for such a script, all modern IDEs offer search capabilities...

**Comment by C-Lodder** [comment]
@dgrammatiko Not to just search for `J` prefixed classes, but to add `use Joomla\xxx\xxx` to the top of the file aswell

File: build/stubGenerator.php [hunk 2]
```diff
@@ -31,6 +31,8 @@
 error_reporting(E_ALL);
 ini_set('display_errors', 1);
 
+use Joomla\CMS\Factory;
```
Comment (Quy) [inline build/stubGenerator.php]: Move after line 13.

File: cli/finder_indexer.php [hunk 3]
```diff
@@ -38,8 +38,10 @@
 // Get the framework.
 require_once JPATH_BASE . '/includes/framework.php';
 
+use Joomla\CMS\Factory;
```
Comment (Quy) [inline cli/finder_indexer.php]: Move line after 23.

File: build/stubGenerator.php (hunk 2 above)
Comment (C-Lodder) [reply build/stubGenerator.php]: done, thanks

File: cli/finder_indexer.php (hunk 3 above)
Comment (C-Lodder) [reply cli/finder_indexer.php]: done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1b107f5dcb3d2c5440ed3f5c4969c8ad6a807753<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24106">issues.joomla.org/tracker/joomla-cms/24106</a>.</sub>

**Comment by amitranjan2** [comment]
I have tested this item :white_check_mark: successfully on 1b107f5dcb3d2c5440ed3f5c4969c8ad6a807753<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24106">issues.joomla.org/tracker/joomla-cms/24106</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by wilsonge** [comment]
Thanks :)

**Pull request merged**